zuul.d/nodesets.yaml

```diff
@@ -959,3 +959,22 @@
         nodes:
           - Trendmicro-DeepSecurity-20.0.198
           - ubuntu-bionic
+
+- nodeset:
+    name: SEP-14-MP1-python38
+    nodes:
+      - name: ubuntu-bionic
+        label: ubuntu-bionic-1vcpu
+      - name: SEP-14-MP1
```

Comment:
It should be `Symantec-Endpoint-Protection-SEP-14-MP1`, See: https://dashboard.zuul.ansible.com/t/ansible/labels